docs/0.1/creating_locations.md

```diff
@@ -0,0 +1,376 @@
+This file sets up the nodes `alpha-node-000` and `beta-node-000` and runs the
+Grid and Splinter components in separate containers; these names appear in
+example prompts and commands. (See [Running Hyperledger Grid on
+Splinter]({% link docs/0.1/grid_on_splinter.md %}) for more information.)
+
+If you are not using this example environment, replace these items with the
+actual values for your environment when entering each command.
+
+### Connect to a Grid Node
+
+1. Connect to the first node's `gridd` container and start a bash session.
+
+   For the example environment, use this command to connect to the `gridd-alpha`
+   container and run Grid commands on `alpha-node-000`.
+
+   ```
+   $ docker exec -it gridd-alpha bash
+   root@gridd-alpha:/#
+   ```
+
+### Set Up Your Environment
+
+Set the following Grid environment variables to specify information for the
+`grid` commands in this procedure.
+
+{:start="2"}
+
+2. Set `GRID_DAEMON_KEY` to the base name of your public/private key files
+   (the example environment uses `alpha-agent`).  This environment variable
+   replaces the `-k` option on the `grid` command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_DAEMON_KEY="alpha-agent"
+   ```
+
+   **Note**: Although this variable has "daemon" in the name, it should
+   reference the ***user key files*** in `$HOME/.grid/keys`, not the Grid
+   daemon's key files in `/etc/grid/keys`.
+
+1. Set `GRID_DAEMON_ENDPOINT` to the Grid daemon's endpoint (URL and port),
+   such as `http://localhost:8080`. This environment variable replaces the
+   `-url` option on the `grid` command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_DAEMON_ENDPOINT="http://localhost:8080"
+   ```
+
+1. Set `GRID_SERVICE_ID` to the fully qualified service ID (such as
+   `01234-ABCDE::gsAA`) for the scabbard service on the circuit. This
+   environment variable replaces the `--service_id` option on the `grid`
+   command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_SERVICE_ID="01234-ABCDE::gsAA"
+   ```
+
+   Tip: See [Prerequisites](#prerequisites) for the `splinter` commands that
+   display the circuit ID and four-character service ID string.
+
+### Define and Create a Location
+
+Each location requires an organization ID (the owner) and at least one agent
+with permission to create, update, and delete the location. There must also
+be an existing location schema for the location's properties.
+See [Prerequisites](#prerequisites) for more information.
+
+{:start="5"}
+
+5. Create a location definition file, in YAML format, that specifies the
+   following information:
+
+   * location namespace (GS1 for this example) that corresponds with the
+     location schema name (`gs1_location` for this example).
+   * Location ID (such as a GTIN)
+   * Owner (organization ID)
+   * Set of location attributes (each with a name and value that conforms to
+     the data type)
+     <br><br>
+
+    ```yaml
+    - namespace: "GS1"
+      location_id: "9012345000004"
```

Comment:
`location_id` and `owner`should be updated to real values to match opened PR#168

```
  location_id: "0013600000011"
  owner: "013600"
```

Reply:
updated